code/modules/mapping/map_template.dm

```diff
@@ -82,6 +82,12 @@
 
 	SSatoms.InitializeAtoms(areas + turfs + movables, returns_created_atoms ? created_atoms : null)
 
+	for(var/turf/unlit as anything in turfs)
+		if(!unlit.always_lit)
```

Comment:
can't you make this an early continue? even the if below can do that

Reply:
I could, I just didn't see this as overly necessary though.